docs/content/bootstrapper.md

```diff
@@ -168,3 +173,10 @@ A few default settings are applied:
 * The bootstrapper is silenced and only errors are displayed. `-v` can be used
   once to restore normal output or twice to show more details.
 * If no version is specified a default `--max-file-age` of `12` hours is used.
+
+**IMPORTANT PERFOMANCE NOTE:**
+
+If you have aggresive Anti Virus monitoring, but have your development folder
```

Comment:
`aggressive`?

Reply:
Where it scans activity of processes in a non-excluded directory. Performance really suffers then if loads of files are created (which paket does a lot :) )

Reply:
But yeah, I agree, it could be worded differently, or even left out.

Reply:
was taking about the typo ;)

Reply:
I hope the updated wording is better - as well as having fewer typos... ;)